data_adapter_oemof/adapters.py

```diff
@@ -400,6 +402,59 @@ class VolatileAdapter(Adapter):
     facade = facades.Volatile
 
 
+class MIMOAdapter(Adapter):
+    """
+    MIMOAdapter
+    """
+
+    type = "mimo"
+    facade = MIMO
+    extra_fields = (
+        Field(name="name", type=str),
+        Field(name="region", type=str),
+        Field(name="year", type=int),
+        Field(name="groups", type=dict),
+        Field(name="capacity_cost", type=dict),
```

Comment:
What is the new type dict used for and why you assigned capacity_cost as dict? As long as it is not a sequence it has no impact really?

Reply:
I think this is a relict from some testing/debugging on my side.
Thanks for pointing out. I think this can be removed safely.